AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DirectionalLightAdded.py

```diff
@@ -89,7 +266,9 @@ def AtomEditorComponents_DirectionalLight_AddedToEntity():
 
     from editor_python_test_tools.editor_entity_utils import EditorEntity
     from editor_python_test_tools.utils import Report, Tracer, TestHelper
-    from Atom.atom_utils.atom_constants import AtomComponentProperties
+    from Atom.atom_utils.atom_constants import AtomComponentProperties, DIRECTIONAL_LIGHT_INTENSITY_MODE, \
```

Comment:
nitpick: we should use `()` instead of `\`

```
from Atom.atom_utils.atom_constants import (
    AtomComponentProperties, DIRECTIONAL_LIGHT_INTENSITY_MODE, DIRECTIONAL_LIGHT_SHADOW_FILTER_METHOD)
```

Reply:
fixed